src/gluonts/mx/model/deepar/_network.py

```diff
@@ -789,6 +791,24 @@ def unroll_encoder_default(
         # static_feat: (batch_size, num_features + prod(target_shape))
         return outputs, state, scale, static_feat, sequence
 
+    def post_process_samples(self, F, samples: Tensor) -> Tensor:
+        """
+        Method to enforce domain-specific constraints on the generated samples.
+        For example, we can enforce forecasts to be nonnegative.
+        Parameters
+        ----------
+        samples
+            Tensor of shape (batch_size * num_samples, 1)
+        Returns
+        -------
+            Tensor of samples with the same shape.
+        """
+
+        if self.nonnegative_pred_samples:
+            return F.Activation(samples, act_type="relu")
```

Comment:
I'm wondering if this works both in symbolic vs eager mode

```suggestion
            return F.relu(samples)
```